helmfile.d/grafana.yaml

```diff
@@ -9,5 +9,3 @@ releases:
         - "../config/default/grafana.yaml"
       secrets:
         - "../secrets/config/grafana/secrets.yaml"
-bases:
```

Comment:
this bit shouldn't really have been reverted, parents inheriting by default is deprecated:
https://github.com/roboll/helmfile/issues/688